app/services/albums.js

```diff
@@ -0,0 +1,28 @@
+const request = require('request-promise');
+const errors = require('errors.js');
+const logger = require('/logger/index.js');
+
+const options = {
+  uri: process.env.ALBUMS_API_URL,
+  headers: { 'User-Agent': 'Request-Promise' },
+  json: true
+};
+
+const get = data => {
+  logger.info(`GET ${data}`);
+  request(data).catch(error => {
+    errors.defaultError(error);
```

Comment:
Mm default error seems too generic, use something like `externalApiError` or something like that